sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -775,10 +772,14 @@ private void finishConfiguration(boolean configureFromApp) {
         }
         mChannel.setEnabled(enabled);
         mChannel.addGroup(CORE_GROUP, DEFAULT_TRIGGER_COUNT, DEFAULT_TRIGGER_INTERVAL, DEFAULT_TRIGGER_MAX_PARALLEL_REQUESTS, null, null);
-        if (mLogUrl != null) {
+        if (mLogUrl != null && mAppSecret != null) {
```

Comment:
you can move this to the other if to combine the checks and code to simplify

```
if (mLogUrl != null) {
    if (mAppSecret != null) {
            mChannel.setLogUrl(mLogUrl);
    }
    else {
           mOneCollectorChannelListener.setLogUrl(mLogUrl);
    }
}
```

In line 779